source/java/filters.rst

```diff
@@ -0,0 +1,32 @@
+.. include:: /includes/pushed-filters.rst
+
+You can use `Java Aggregation Expressions <https://www.mongodb.com/docs/drivers/java/sync/upcoming/fundamentals/aggregation-expression-operations/>`__ to filter your data.
```

Comment:
S: use an rst target such as the [driver](https://github.com/mongodb/snooty-parser/blob/edc552dca3972874ab566434d14b2f13cc749464/snooty/rstspec.toml#L1428C6-L1430) one